docs/containers-registries.conf.5.md

```diff
@@ -48,7 +48,38 @@ registries = ['registry3.com']
 registries = ['registry.untrusted.com', 'registry.unsafe.com']
 ```
 
+## VERSION 2
+In addition to the initial format a second one can be used, which is capable in
+providing additional registry mirrors.  Every registry can have their own
+mirrors, where it is possible to set the `insecure` option per mirror.  The
+mirrors will be evaluated in order during an image pull.
+
+Furthermore it is possible to specify a `prefix` for a registry. The `prefix` is
+used for matching images, and to translate one namespace to another.  If
+`prefix = "example.com/bar"`, `url = "example.com/foo/bar"` and the image will be
+pulled from `example.com/bar/myimage:latest`, the image will effectively be
+pulled from `example.com/foo/bar/myimage:latest`.  If no prefix is specified,
+it defaults to the specified `url`. If a prefix is specified, direct pull via
+the `url` above is not possible any more.
+
```

Comment:
I don' think the description suffices for new users to really understand what can be configured and the semantics of each switch and option.

Explaining those in detail may be a bit redundant to the previous section but I think it should be self-containing.

Maybe, we can mention the new format first and mention the old one below (but without mentioning the version)? @mtrmac WDYT?

Reply:
> As for which one goes first — the best one I guess :) I think all GitHub.com/containers consumers now support v2, don’t they?

Yes, they do.

Reply:
I updated the documentation and made the example more clear.